src/pages/NewBeerPage.jsx

```diff
@@ -0,0 +1,151 @@
+import axios from "axios";
+import React, { useState } from "react";
+import { useNavigate } from "react-router-dom";
+import Header from "../components/Header";
+ 
+function NewBeerPage() {
+  const [name, setName] = useState("");
+  const [tagline, setTagline] = useState("");
+  const [description, setDescription] = useState("");
+  const [firstBrewed, setFirstBrewed] = useState("");
+  const [brewerTips, setBrewerTips] = useState("");
+  const [attenuationLevel, setAttenuationLevel] = useState(1);
+  const [contributedBy, setContributedBy] = useState("");
+
+  const navigate = useNavigate();
+
+  const handleSubmit = async (e) => {
+    e.preventDefault();
+
+    const body = {name, description, tagline, first_brewed: firstBrewed, brewers_tip: brewerTips, contributed_by: contributedBy, attenuation_level:attenuationLevel}
+    try{
+      await axios.post("https://ih-beers-api2.herokuapp.com/beers/new", body)
+      navigate("/")
```

Comment:
Maybe here I would send the user to the list of all the beers, so they can see the beer created